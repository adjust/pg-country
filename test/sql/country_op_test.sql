-- os_name_op should implement gt;
-- ./spec/country_op_spec.rb:8;
BEGIN;
CREATE EXTENSION country;
SELECT 'uk'::country > 'de'::country;
SELECT 'uk'::country >= 'de'::country;
SELECT NOT('uk'::country < 'de'::country);
SELECT NOT('uk'::country <= 'de'::country);
ROLLBACK;
-- os_name_op should implement lt;
-- ./spec/country_op_spec.rb:15;
BEGIN;
CREATE EXTENSION country;
SELECT 'uk'::country > 'de'::country;
SELECT 'uk'::country >= 'de'::country;
SELECT NOT('uk'::country < 'de'::country);
SELECT NOT('uk'::country <= 'de'::country);
ROLLBACK;
-- os_name_op should implement eq;
-- ./spec/country_op_spec.rb:22;
BEGIN;
CREATE EXTENSION country;
SELECT NOT('de'::country = 'uk'::country);
SELECT 'uk'::country = 'uk'::country;
ROLLBACK;
-- os_name_op should implement neq;
-- ./spec/country_op_spec.rb:27;
BEGIN;
CREATE EXTENSION country;
SELECT 'de'::country != 'uk'::country;
SELECT NOT('uk'::country != 'uk'::country);
ROLLBACK;
