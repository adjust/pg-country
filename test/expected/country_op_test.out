-- os_name_op should implement gt;
-- ./spec/country_op_spec.rb:8;
BEGIN;
CREATE EXTENSION country;
SELECT 'uk'::country > 'de'::country;
 ?column? 
----------
 t
(1 row)

SELECT 'uk'::country >= 'de'::country;
 ?column? 
----------
 t
(1 row)

SELECT NOT('uk'::country < 'de'::country);
 ?column? 
----------
 t
(1 row)

SELECT NOT('uk'::country <= 'de'::country);
 ?column? 
----------
 t
(1 row)

ROLLBACK;
-- os_name_op should implement lt;
-- ./spec/country_op_spec.rb:15;
BEGIN;
CREATE EXTENSION country;
SELECT 'uk'::country > 'de'::country;
 ?column? 
----------
 t
(1 row)

SELECT 'uk'::country >= 'de'::country;
 ?column? 
----------
 t
(1 row)

SELECT NOT('uk'::country < 'de'::country);
 ?column? 
----------
 t
(1 row)

SELECT NOT('uk'::country <= 'de'::country);
 ?column? 
----------
 t
(1 row)

ROLLBACK;
-- os_name_op should implement eq;
-- ./spec/country_op_spec.rb:22;
BEGIN;
CREATE EXTENSION country;
SELECT NOT('de'::country = 'uk'::country);
 ?column? 
----------
 t
(1 row)

SELECT 'uk'::country = 'uk'::country;
 ?column? 
----------
 t
(1 row)

ROLLBACK;
-- os_name_op should implement neq;
-- ./spec/country_op_spec.rb:27;
BEGIN;
CREATE EXTENSION country;
SELECT 'de'::country != 'uk'::country;
 ?column? 
----------
 t
(1 row)

SELECT NOT('uk'::country != 'uk'::country);
 ?column? 
----------
 t
(1 row)

ROLLBACK;
